src/Penrose/GenOptProblem.hs

```diff
@@ -1006,7 +1006,18 @@ genOptProblemAndState trans optConfig =
 -- NOTE: this function also print information out to stdout
 -- TODO: enable logger
 compileStyle :: StyProg -> C.SubOut -> [J.StyVal] -> OptConfig -> IO State
-compileStyle styProg (C.SubOut subProg (subEnv, eqEnv) labelMap) styVals optConfig = do
+compileStyle styProgInit (C.SubOut subProg (subEnv, eqEnv) labelMap) styVals optConfig = do
+   putStrLn "Parsed Style program\n"
```

Comment:
We need to get rid of all print statements (at lease the ones to stdout) for the Automator to work properly, as it communicates with the backend binary via stdout and stdin

Reply:
Does this mean that all the print statements in `compileStyle` need to go? There are a lot of them